projects/ffmpeg/build.sh

```diff
@@ -315,40 +320,6 @@ fuzzer_name=ffmpeg_IO_DEMUXER_fuzzer
 make tools/target_io_dem_fuzzer
 mv tools/target_io_dem_fuzzer $OUT/${fuzzer_name}
 
-#Build fuzzers for individual demuxers
```

Comment:
@michaelni can I ask if this is safe to do?

Reply:
The reason for the multiple configure IIRC was to make the binaries as small as possible because we did in the past hit diskspace constraints on google infrastructure.

Reply:
Thanks! Indeed it looks like it increases the size of the dem binaries from 34MB each to 56MB (but not the other binaries). I think we can make it work with both ./configures, so I've restore it.